app/views/shared/_batch.html.erb

```diff
@@ -1,5 +1,5 @@
 
-  <table class="sortable table table-stripped">
```

Comment:
Typo here, so the striped style wasn't applied. Decided to just remove it, rather than ad striping.